src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -34,6 +34,9 @@ import { venueLandingUrl } from "utils/url";
 import { RoomsForm } from "pages/Admin/Venue/Rooms/RoomsForm";
 import { VenueAdminPage } from "pages/Admin/Venue/VenueAdminPage";
 
+// TODO: Add a route/some kind of related functionality here or similar for handling receiving the custom auth token back from firebase, then using that to sign in.
+//   Alternatively, should we handle this in src/index.ts?
+
```

Comment:
[TODO] Add a route/some kind of related functionality here or similar for handling receiving the custom auth token back from firebase

See other comments in this review for thoughts on what to use for the route here.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1523/commits/bba51569720b6b1215b83b06e481f811d677ed52

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1523/commits/bba51569720b6b1215b83b06e481f811d677ed52